packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -595,7 +596,7 @@ export interface SdkBasicServiceMethod<TServiceOperation extends SdkServiceOpera
 }
 
 interface SdkPagingServiceMethodOptions {
-  __raw_paged_metadata: PagedResultMetadata;
+  __raw_paged_metadata?: PagedResultMetadata;
```

Comment:
Any reason why this become optional?

Reply:
unbranded part does not have this metadata. since it is not recommended to use raw, i think such breaking is acceptable. looks like it only affect .net, i'll let them to change the usage.